content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-sample.md

```diff
@@ -481,29 +468,29 @@ In this step, we create the `UsageCostLogger` sink.
 
 <!--Kafka-->
 
-You can either [directly download the project generated by the Spring Initialzr](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.4.3.RELEASE&packaging=jar&jvmVersion=1.8&groupId=io.spring.dataflow.sample&artifactId=usage-cost-logger-kafka&name=usage-cost-logger-kafka&description=Demo%20project%20for%20Spring%20Boot&packageName=io.spring.dataflow.sample.usagecostlogger&dependencies=cloud-stream,actuator,kafka)
+You can either [directly download the project generated by the Spring Initialzr](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.4.3.RELEASE&packaging=jar&jvmVersion=1.8&groupId=io.spring.dataflow.sample&artifactId=usage-cost-logger-kafka&name=usage-cost-logger-kafka&description=Demo%20project%20for%20Spring%20Boot&packageName=io.spring.dataflow.sample.usagecostlogger&dependencies=cloud-stream,actuator,kafka) and click on **Generate Project**.
 
 or visit the [Spring Initialzr site](https://start.spring.io/) and follow these instructions:
 
 1. Create a new Maven project with a Group name of `io.spring.dataflow.sample` and an Artifact name of `usage-cost-logger-kafka`, and package `o.spring.dataflow.sample.usagecostlogger`.
-2. In the **Dependencies** text box, type `Kafka` to select the Kafka binder dependency,.
+2. In the **Dependencies** text box, type `Kafka` to select the Kafka binder dependency.
 3. In the **Dependencies** text box, type `Cloud Stream` to select the Spring Cloud Stream dependency.
 4. In the **Dependencies** text box, type `Actuator` to select the Spring Boot actuator dependency.
-5. Click the **Generate Project** button.
+5. Click on **Generate Project**.
 
 Now you should `unzip` the `usage-cost-logger-kafka.zip` file and import the project into your favorite IDE.
 
 <!--RabbitMQ-->
 
-You can either [directly download the project generated by the Spring Initialzr](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.4.3.RELEASE&packaging=jar&jvmVersion=1.8&groupId=io.spring.dataflow.sample&artifactId=usage-cost-logger-rabbit&name=usage-cost-logger-rabbit&description=Demo%20project%20for%20Spring%20Boot&packageName=io.spring.dataflow.sample.usagecostlogger&dependencies=cloud-stream,actuator,amqp)
+You can either [directly download the project generated by the Spring Initialzr](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.4.3.RELEASE&packaging=jar&jvmVersion=1.8&groupId=io.spring.dataflow.sample&artifactId=usage-cost-logger-rabbit&name=usage-cost-logger-rabbit&description=Demo%20project%20for%20Spring%20Boot&packageName=io.spring.dataflow.sample.usagecostlogger&dependencies=cloud-stream,actuator,amqp) and click on **Generate Project**.
```

Comment:
As mentioned above you can add `starter.zip?` at the beginning of the URL and the user will get a download, instead of going to the site.